.github/workflows/deploy_hold.yml

```diff
@@ -27,14 +27,14 @@ jobs:
     runs-on: ubuntu-latest
     environment: production # sets in github repo with reviewer requirement protection rule
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Download build-output-US artifact
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4
```

Comment:
have you checked that the newly uploaded artifacts in aws and gcp are the same format as before?

Reply:
for aws, seems like although [the new file size](https://github.com/yext/answers-search-ui/actions/runs/9024459183) of `build-output-US` is ~8mb, the files in its unzipped folder accumulate to ~39mb, and look same as they are in the current [aws bucket ](https://us-east-1.console.aws.amazon.com/s3/buckets/assets.sitescdn.net?region=us-east-1&bucketType=general&prefix=answers/dev/dev-update-workflow/&showversions=false) (after logging in). not sure why the artifact sizes in the workflow screens are [different](https://github.com/yext/answers-search-ui/actions/runs/8694652127). my guess is that github is now using some new kind of zipping algorithm? 


![Screenshot 2024-05-09 at 11 33 12 PM](https://github.com/yext/answers-search-ui/assets/12143814/2bf74487-5dbf-4edc-9827-5bd5f3808c37)

Reply:
ok, so after unzipping both the old and new artifacts, the folders have the same size? if so, that seems fine. also, have you tested by spinning up the Theme test site and pointing to the new assets you deployed to aws? you'll want to spot check to make sure everything still works as expected using them